tests/integration_tests/with_db/udfs/test_translation_script.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from tests.utils.parameters import model_params
+
+
+@pytest.mark.parametrize("upload_model_to_bucketfs", [model_params.seq2seq],
+                         indirect=["upload_model_to_bucketfs"])
+def test_translation_script(
+        upload_language_container, setup_database,
+        pyexasol_connection, upload_model_to_bucketfs):
+
+    bucketfs_conn_name, schema_name = setup_database
+    n_rows = 100
+    model_name = "t5-small"
+    src_lang = "English"
+    tgt_lang = "German"
+    max_length = 50
```

Comment:
why do need max length for translation. Exists this parameter, really.

Reply:
It is used in almost all translation examples. If the length is short, it summarizes the given text accordingly. I thought it might be useful.